src/reference/glossary.rst

```diff
@@ -12,10 +12,16 @@ Glossary
 
       A web application used for phylogenetic visualization and analysis. :doc:`Documentation<auspice:index>`
 
+   pathogen repository
+
+      A version-controlled folder containing all files necessary to run a pathogen's :term:`workflows<workflow>`.
+
    workflow
-      also *pathogen workflow*, *pathogen analysis*, *Nextstrain workflow*
```

Comment:
The tutorial titles "Running a pathogen workflow" / "Creating a pathogen workflow" no longer have a clear definition of "pathogen workflow". The definition changes are getting rid of the previously assumed mapping of 1 repo : 1 workflow which is also reflected in changes to the zika repo. (Suggestion 1) This would be a good opportunity to replace zika-tutorial with zika as mentioned elsewhere.

(Suggestion 2) Rework the tutorials to account for the multi-workflow repo:

> Title: Using a pathogen repository
> 
> 1. Clone zika repo
> 2. Explain the different workflows
> 3. Assume ingest/nextclade are not necessary and show how to run the phylo workflow (existing content)

> Title: Creating a pathogen repository
> 
> 1. Show how to run a build by calling Augur directly (existing content)
> 2. Explain the scoping of `phylogenetic/`

Maybe it'd be useful to have an additional tutorial for users who want to run without the rigid pathogen repo layout, e.g. "Running a phylogenetic analysis with Augur commands and visualizing with Auspice", where contents would be what's in the current "Creating a pathogen workflow" tutorial.

Reply:
Thanks! I think both suggestions fit into the long term goal to have a thorough [docs for creating a pathogen repo](https://github.com/nextstrain/docs.nextstrain.org/issues/188). 

I think I'd just rename the two tutorial titles to "Running a phylogenetic workflow" and "Creating a phylogenetic workflow" but incorporate the other suggestions in future work. I think the addition of the ingest tutorials will require some restructuring of the existing tutorials anyways.

Reply:
Updated in cb76eebaa60a0a59d8549be264c9260e12254a25